apps/appkit-minter/.env.example

```
@@ -0,0 +1 @@
+WALLET_MNEMONIC=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)